### PR TITLE
added changes to improve entities_utilities

### LINKED ISSUE #10308: [GeoMechanicsApplication] Edited EnitityUtility to improve control entities that are active in initialization and finalization
Changes to EntitieUtility to control better the initialization and finalization

### PR DESCRIPTION
**Pull request by mcgicjn2**
**📝 Description**
Improving the initialization and finalization of entities to only include those that entities that are active during the solutionstep. Originally implemented by @Vahid-Galavi.

**🆕 Changelog**
- Added active check to entitie for initialize and finalize solution step.

### CODE REVIEW

File: kratos/utilities/entities_utilities.h [hunk 1]
```diff
@@ -118,7 +118,16 @@ namespace EntitiesUtilities
...
             GetEntities<TEntityType>(rModelPart),
             [&r_current_process_info](TEntityType& rEntity){
-                rEntity.InitializeSolutionStep(r_current_process_info);
+                // Detect if the entity is active or not. If the user did not make any choice the entity
+                // It is active by default
+                const bool entity_is_active = (rEntity.IsDefined(ACTIVE)) ? rEntity.Is(ACTIVE) : true;
+                KRATOS_INFO("element") << rEntity.Id() 
+                        << " IsDefined: " << rEntity.IsDefined(ACTIVE) 
+                        << " Set: "       << rEntity.Is(ACTIVE) << std::endl;
+
```
Comment (rubenzorrilla) [inline kratos/utilities/entities_utilities.h]:
```suggestion
```
This will be extremely verbose.

Comment (loumalouomega) [reply kratos/utilities/entities_utilities.h]: Indeed

**Review by RiccardoRossi — CHANGES_REQUESTED** [review]
the current behaviour is consistent with the one of the builder and solver, so i am afraid we cannot change it easily.

@KratosMultiphysics/technical-committee opinions?

**Comment by mcgicjn2** [comment]
As this is a stale branch and the functionality appears not to be required on our side. Perhaps we can close this and related branches I.e. #10265

**Comment by mcgicjn2** [comment]
Closing

**Pull request closed without merging**